package.json

```diff
@@ -42,38 +42,37 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "^7.6.4",
-    "@babel/preset-env": "^7.6.3",
-    "@wordpress/babel-preset-default": "^3.0.2",
-    "@wordpress/eslint-plugin": "^1.0.1",
-    "autoprefixer": "^9.6.4",
+    "@babel/core": "^7.9.0",
+    "@babel/preset-env": "^7.9.5",
+    "@wordpress/babel-preset-default": "^4.12.1",
+    "@wordpress/eslint-plugin": "^5.0.1",
+    "autoprefixer": "^9.7.6",
     "babelify": "^10.0.0",
-    "eslint": "^5.16.0",
+    "eslint": "^6.8.0",
     "grunt": "^1.0.4",
     "grunt-browserify": "^5.3.0",
     "grunt-contrib-clean": "^2.0.0",
     "grunt-contrib-concat": "^1.0.1",
     "grunt-contrib-copy": "^1.0.0",
-    "grunt-contrib-uglify": "^3.3.0",
+    "grunt-contrib-uglify": "^4.0.1",
     "grunt-contrib-watch": "^1.1.0",
-    "grunt-modernizr": "^1.0.3",
+    "grunt-modernizr": "^3.0.0",
     "grunt-notify": "^0.4.1",
-    "grunt-phplint": "~0.1.0",
     "grunt-postcss": "^0.9.0",
     "grunt-sass": "^3.1.0",
     "grunt-sass-lint": "^0.2.4",
-    "grunt-version": "^1.3.1",
+    "grunt-version": "^2.0.0",
     "grunt-wp-i18n": "^1.0.2",
-    "node-sass": "^4.12.0",
-    "time-grunt": "^1.4.0"
+    "node-sass": "^4.14.0",
+    "time-grunt": "^2.0.0"
   },
   "scripts": {
     "postinstall": "grunt install"
   },
   "dependencies": {
     "browserify-shim": "^3.8.14",
     "lg-thumbnail": "^1.1.0",
-    "lightgallery": "^1.6.11",
+    "lightgallery": "^1.6.14",
     "responsive-foundation": "git+ssh://git@github.com:bu-ist/responsive-foundation#3.2.3"
```

Comment:
Isn't foundation up to 4.0?

Reply:
It is, but I'm holding off on upgrading Framework (so that all sites across BU use it) because there was a major change to the grid. I'm hoping the WordPress upgrade visual regression tool will make it possible for me to upgrade this more confidently. Anywho, it's not unusual (for this reason) for Framework to be a couple of versions behind Foundation - usually the way we roll major changes out is child themes get it first, we work out all the bugs with child themes, and then Framework gets it once we feel it's really really stable. I think we're at that point now, but I'm worried about the testing, so I'm holding back for now.